Add MCP server for Redpanda docs (#117)
redpanda-data/docs-site: home/modules/ROOT/pages/mcp-setup.adoc

> = MCP Remote Server

Author: ```suggestion
= MCP Remote Server for Redpanda Documentation
```

> :description: Learn how to connect to the Redpanda documentation MCP server in Cursor, VSCode, and other AI tools.

Reviewer: ```suggestion
:description: Learn how to connect to the Redpanda documentation MCP server in Cursor, VS Code, and other AI tools.
```

> Redpanda provides a remote *Model Context Protocol (MCP)* server that lets you access documentation directly from your IDE or AI tool such as Cursor, VSCode or Claude.

Author: ```suggestion
Redpanda provides a remote link:https://modelcontextprotocol.io[*Model Context Protocol (MCP)* server^] that lets you access documentation directly from your IDE or AI tool such as Cursor, VSCode or Claude.
```

> Redpanda provides a remote link:https://modelcontextprotocol.io[Model Context Protocol (MCP) server^] that lets you access documentation directly from your IDE or AI tool such as Cursor, VSCode or Claude.

Reviewer: ```suggestion
Redpanda provides a remote link:https://modelcontextprotocol.io[Model Context Protocol (MCP) server^] that lets you access documentation directly from your IDE or AI tool, such as Cursor, VSCode, or Claude.
```

The MCP server is hosted at: `https://docs.redpanda.com/mcp`.
You can add this endpoint to any AI agent that supports MCP.

== Set up

[tabs]
====
Cursor::
+
--
link:cursor://mcp/add?name=redpanda&url=https://docs.redpanda.com/mcp[Automatically add the Redpanda MCP server to Cursor].

Or, manually add it by editing your `.cursor/mcp.json` file, as explained in the https://docs.cursor.com/context/model-context-protocol[Cursor documentation^]:

[source,json]
----
{
  "mcpServers": {
    "redpanda": {
      "url": "https://docs.redpanda.com/mcp"
    }
  }
}
----
--

> VSCode::
> +
> --
> link:https://vscode.dev/redirect/mcp/install?name=redpanda&config=%7B%22type%22%3A%22http%22%2C%22url%22%3A%22https%3A%2F%2Fdocs.redpanda.com%2Fmcp%22%7D[Automatically add the Redpanda MCP server to VSCode].

Reviewer: ```suggestion
VS Code::
+
--
link:https://vscode.dev/redirect/mcp/install?name=redpanda&config=%7B%22type%22%3A%22http%22%2C%22url%22%3A%22https%3A%2F%2Fdocs.redpanda.com%2Fmcp%22%7D[Automatically add the Redpanda MCP server to VS Code].
```

Or, manually create a `.vscode/mcp.json` file in your workspace:

.`.vscode/mcp.json`
[source,json]
----
{
  "servers": {
    "redpanda": {
      "url": "https://docs.redpanda.com/mcp"
    }
  }
}
----

To configure globally, run **MCP: Open User Configuration** from the Command Palette and add the same JSON to the `mcp` section.

> For more, see the https://code.visualstudio.com/docs/copilot/chat/mcp-servers[VSCode MCP documentation^].

Reviewer: ```suggestion
For more, see the https://code.visualstudio.com/docs/copilot/chat/mcp-servers[VS Code MCP documentation^].
```

--
Claude Desktop::
+
--
To add Redpanda's MCP server to Claude Desktop:

**On macOS:**
`~/Library/Application Support/Claude/claude_desktop_config.json`

**On Windows:**
`%APPDATA%\Claude\claude_desktop_config.json`

Add this configuration:

[source,json]
----
{
  "mcpServers": {
    "redpanda": {
      "url": "https://docs.redpanda.com/mcp"
    }
  }
}
----

Restart Claude Desktop for changes to take effect.
--
====

== Other AI tools

Any tool that supports MCP servers can connect using the following URL:

[source,text]
----
https://docs.redpanda.com/mcp
----

Popular tools that support MCP:

> * https://github.com/microsoft/vscode-copilot[GitHub Copilot^] (VSCode 1.102+)

Reviewer: ```suggestion
* https://github.com/microsoft/vscode-copilot[GitHub Copilot^] (VS Code 1.102+)
```
